### PR TITLE
disable crouch if player is not on the ground

### PR DESCRIPTION
**Pull request by tcm390**
In this PR, disable crouch when player is not on the ground

related:
https://github.com/webaverse/app/issues/3529

https://user-images.githubusercontent.com/60634884/183292509-0ed675ee-46be-4776-8264-9a56855ff2d3.mp4

### CODE REVIEW

File: io-manager.js [hunk 1]
```diff
@@ -245,12 +245,13 @@ const _updateIo = timeDiff => {
       keysDirection.applyQuaternion(camera.quaternion);
       _updateVertical(keysDirection);
     } else {
+      const isGrounded = localPlayer.characterPhysics.lastGrounded;
```
Comment (avaer) [inline io-manager.js:248]: We should not re-use tracking flags like this. If you want to track if the character is grounded, we should have an API for that.

**Review by avaer — CHANGES_REQUESTED** [review]
Recomending API refactor.

File: io-manager.js (hunk 1 above)
Comment (tcm390) [reply io-manager.js]: [add isGrounded API in game.js](https://github.com/webaverse/app/pull/3543/commits/f7db09a71021d68980c8022850eaf722daf85f3a)

**Comment by alisaad673** [comment]
Working fine.

**Review by avaer — APPROVED** [review]

**Pull request merged**